laa_crime_application_store_app/routers/v1/application.py

```diff
@@ -102,4 +107,45 @@ async def put_application(
         return Response(status_code=409)
 
     logger.info("APPLICATION_UPDATED", application_id=application.application_id)
+    NotificationService().notify(db, request, application.application_id)
     return Response(status_code=201)
+
+
+@router.post("/subscriber")
+@auth_logger
+async def post_subscriber(
+    request: Request,
+    subscriber: Subscriber,
+    db: Session = Depends(get_db),
+):
+    logger.info("CREATING_SUBSCRIBER")
+    existing_subscriber = NotificationService.subscribe(
+        db, subscriber.subscriber_type, subscriber.webhook_url
+    )
+
+    if existing_subscriber is None:
+        logger.info("SUBSCRIBER_CREATED")
+        return Response(status_code=201)
+    else:
+        logger.info("SUBSCRIBER_ALREADY_EXISTED")
+        return Response(status_code=204)
+
+
+@router.delete("/subscriber")
```

Comment:
should we have an update as well?

Reply:
and maybe put subscriber in its own router so we can do different things with authentication etc if we need to?

Reply:
I can't think of a use case for date at the mo. For branch deployments we create on startup and delete on environment deletion, and beyond that I think maybe it's something we add when we have a need for it. But yes, sounds very sensible to have a different router for subscriber.